containers/Dockerfile.buildenv

```diff
@@ -300,6 +271,56 @@ RUN wget http://mirror.ctan.org/systems/texlive/tlnet/install-tl-unx.tar.gz \
     && /work/texlive/bin/x86_64-linux/tlmgr install bibtex
 ENV PATH="${PATH}:/work/texlive/bin/x86_64-linux"
 
+# Remove the apt-get cache in order to reduce image size
+RUN apt-get -y clean
+
+
+# Everyhting we need to run tests on CI on GitHub.
```

Comment:
Everything

Reply:
Why am I so bad at spelling 😓

Reply:
https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker 😉

Reply:
What makes it even worse is that I actually have that extension enabled and still missed it. 😅  (To be fair, it got lost in all of the "incorrect" spellings of library names and bash commands)